python/semantic_kernel/connectors/telemetry.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import os
+
+import pkg_resources
+
+TELEMETRY_DISABLED_ENV_VAR = "AZURE_TELEMETRY_DISABLED"
+
+IS_TELEMETRY_ENABLED = (
+    os.environ.get(TELEMETRY_DISABLED_ENV_VAR, "false").lower() != "true"
```

Comment:
for consistency with the C#/Azure logic, this should check also for "1"

Reply:
```suggestion
IS_TELEMETRY_ENABLED = os.environ.get(
    TELEMETRY_DISABLED_ENV_VAR, "false"
).lower() not in ["true", "1"]
```

Reply:
Applied your suggestion